### DISCUSSION TITLE
How can I open a modal popup on Dawn?

### DISCUSSION THREAD

**Discussion by Zeioth** [Shopify/dawn#2288] [Q&A]
I've seen we have the .class product-popup-modal class so I'm guessing we have some kind of popup implemented, but I'm not sure how to use it. Is this documented somewhere? I want to achieve something like

![screenshot_2023-02-06_23-20-41_952848098](https://user-images.githubusercontent.com/3357792/217100700-f2ebe3be-b946-4740-b3c1-8b73a9191b4f.png)

**Comment by ryanm-bw** [comment]
Take a look at `<modal>` and `<modal-opener>` components in `global.js` 

This is a great example of where decent documentation for the individual components would be very helpful.
